src/components/ui/svgs/icons.js

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
```

Comment:
There should never be a need to import React. Generally there is something more precise to import. If it is unused then this line should be removed.